Sample/LuminaSample/ViewController.swift

```diff
@@ -83,9 +83,7 @@ extension ViewController { // MARK: IBActions
       if let map = sender as? [String: Any] {
         controller.image = map["stillImage"] as? UIImage
         controller.livePhotoURL = map["livePhotoURL"] as? URL
-        if #available(iOS 11.0, *) {
-          controller.depthData = map["depthData"] as? AVDepthData
-        }
+        controller.depthData = map["depthData"] as? AVDepthData
         let positionBool = map["isPhotoSelfie"] as! Bool
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/8cefb8053a7a558007ec4a93a1a2b01854a11a38/Sample/LuminaSample/ViewController.swift#L87"></span>Force casts should be avoided.<br />
<code>force_cast</code> <code>ViewController.swift:87</code></td>
    </tr>
  </tbody>
</table>